internal/kafka/internal/services/clusters.go

```diff
@@ -397,7 +400,8 @@ func (c clusterService) FindKafkaInstanceCount(clusterIDs []string) ([]ResKafkaI
 	var kafkas []*dbapi.KafkaRequest
 
 	query := c.connectionFactory.New().
-		Model(&dbapi.KafkaRequest{})
+		Model(&dbapi.KafkaRequest{}).
+		Where("status not in (?)", kafkaStatusesThatNoLongerConsumesResourcesInTheDataPlane)
```

Comment:
Can we update the method documentation to describe this?

Reply:
done

Reply:
:+1: